installer/src/components/AccountForm/AccountForm.tsx

```diff
@@ -0,0 +1,147 @@
+import LoadingSpinner from '../LoadingSpinner/LoadingSpinner';
+import NoPluginMessage from '../NoPluginMessage/NoPluginMessage';
+
+const AccountForm = () => {
+  // Call useEffect to call function to get account Name
+  const [accountName, setAccountName] = useState<string>("");
+  const [accountSid, setAccountSid] = useState<string>("");
+  const [accountAuthToken, setAccountAuthToken] = useState<string>("");
+  const [pluginFriendlyName, setPluginFriendlyName] = useState<string>("");
+  const [pluginSid, setPluginSid] = useState<string>("");
+  const [hlsPluginExists, setHlsPluginExists] = useState<boolean>(true);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const handleDeployment = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    setIsLoading(true);
+    if (accountSid && accountAuthToken) {
+      // This call creates the workspace and routers via APIs
+      await fetch(getBackendUri() + "/task-router/deploy-task-router")
+        .then(resp => resp.json())
+        .then(data => {
+          console.log(data);
+        })
+        .catch(err => console.error(err));
+
+      // This call gets all the plugins
+      const plugin = await fetch(getBackendUri() + "/plugin/get-plugins")
+        .then(plugin => plugin.json())
+        .then(async resp => {
+          const plugins: Plugin[] = resp.data.plugins;
+          const hlsPlugin = Object.values(plugins).find(plugin => plugin.friendly_name === hlsPluginName);
+          if (hlsPlugin) {
+            setPluginFriendlyName(hlsPlugin.friendly_name);
+            setPluginSid(hlsPlugin.sid);
+          }
+          return hlsPlugin;
+        })
+        .catch(err => console.error(err));
+      if (!plugin) setHlsPluginExists(false);
+      
+      // Release the plugin config to Flex Account
+      const configRelease = await fetch(getBackendUri() + "/plugin/release-plugin")
+        .then(res => res.json())
+        .then(release => release.data)
+        .catch(err => console.error(err));
+      if (configRelease) setIsLoading(false);
+      console.log(configRelease);
+    }
+  }
+
+  useEffect(() => {
+    const getAccount = async () => {
+      await fetch(getBackendUri() + "/account/get-account-name")
+      .then(resp => resp.json())
+      .then(jsonResp => setAccountName(jsonResp.data.accountName));
+
+      await fetch(getBackendUri() + "/account/get-account-info")
+        .then(resp => resp.json())
+        .then(async (jsonResp) => {
+          setAccountSid(jsonResp.data.accountSid);
+          setAccountAuthToken(jsonResp.data.authToken);
+      });
+    }
+    getAccount();
+  }, []);
+
+  return (
+    <div className='col-start-5 col-span-4 flex flex-col'>
+      <form className=''>
+        <div className='flex flex-col'>
+          <div className='row mb-5 text-deploy-blue'>
+              <div className='mb-3'>
+                <span className='font-bold pr-2'>Step 1:</span>
```

Comment:
could be some kind of heading element no? like an `<h2 />`? heading elements are especially important for screen readers/SEO to quickly scan through important sections of a page